profiles/common/modules/features/multisite_charts/multisite_charts.module

```diff
@@ -69,15 +69,16 @@ function theme_multisite_chart(array $variables = array()) {
 function multisite_charts_get_library_path() {
   global $conf;
 
-  $path = 'sites/all/libraries/FusionCharts/';
-  $lpath = libraries_get_path('FusionCharts');
   if (isset($conf['fusion_charts_path'])) {
-    $path = $conf['fusion_charts_path'];
+    return $conf['fusion_charts_path'];
   }
-  elseif ($lpath) {
-    $path = $lpath . '/';
+  if (module_exists('libraries')) {
+    $path = libraries_get_path('FusionCharts');
   }
-  return $path;
+  if (!$path) {
+    $path = 'profiles/common/libraries/FusionCharts';
+  }
+  return $path . '/';
```

Comment:
This is not ideal, since this will make the feature unusable outside of the platform. This path `profiles/common/libraries/` is unique to the platform.

I do realize that it is very unlikely that the feature will work outside of the platform anyway, so I will let this slide for now.